config/default.json

```diff
@@ -318,7 +318,10 @@
         "UseExperimentalGossip": false,
         "ReadOnlyConfig": true,
         "GossipPort": 8074,
-        "StreamingPort": 8075
+        "StreamingPort": 8075,
+        "MaxIdleConns": 100,
+        "MaxIdleConnsPerHost": 2,
```

Comment:
We should discuss if we want this to be other than the `http.DefaultTransport`, by default.

Reply:
I bet we want that higher. I don't see any issues with keeping connections open, only benefits.

Reply:
Kubernetes has this set at 25: https://github.com/kubernetes/kubernetes/blob/f2ddd60eb9e7e9e29f7a105a9a8fa020042e8e52/pkg/client/transport/cache.go#L37, though it's hard not to want a power of 2 here. Should we default to 16 and see where that takes us?